apps/src/templates/teacherNavigation/TeacherNavigationRouter.tsx

```diff
@@ -199,7 +210,12 @@ const TeacherNavigationRouter: React.FC<TeacherNavigationRouterProps> = ({
             element={
               <ElementOrEmptyPage
                 showNoStudents={studentCount === 0}
-                showNoCurriculumAssigned={!anyStudentHasProgress}
+                showNoCurriculumAssigned={!selectedSection?.courseOfferingId}
+                showNoCalendarForLegacyCourses={parseInt(versionYear) < 2021}
```

Comment:
```suggestion
                showNoCalendarForLegacyCourses={parseInt(selectedSection?.versionYear) < 2021}
```
Can we just use the `?` instead of a new variable?